RadixWallet/Features/AccountListFeature/Coordinator/AccountList+Reducer.swift

```diff
@@ -6,20 +6,12 @@ public struct AccountList: Sendable, FeatureReducer {
 	public struct State: Sendable, Hashable {
 		public var accounts: IdentifiedArrayOf<AccountList.Row.State>
 
-		public init(accounts: IdentifiedArrayOf<AccountList.Row.State>) {
-			self.accounts = accounts
-		}
-
 		public init() {
-			self.accounts = []
+			self.init(accounts: [])
```

Comment:
Why is `accounts` an identifiable array in the first place? Unless we want to manipulate it directly in this reducer, that's not needed.

Reply:
I think it is for `ForEach`, kept the same definitions.